aten/src/ATen/native/native_functions.yaml

```diff
@@ -6668,3 +6668,10 @@
   variants: function
   device_guard: False
   supports_named_tensor: True
+
+# Note: this function is only for testing.
+# It is undocumented and should not be used outside of tests.
+- func: _subcmul(Tensor self, Tensor other, Scalar alpha=1) -> Tensor
```

Comment:
honestly I'd probably put that in the name.  Something really ugly like _test_serialization_subcmul.

Reply:
Good idea.